bsmschema/bsmschema/models.py

```diff
@@ -160,40 +154,31 @@ class Transformations(_Commentable):
     The format of these instructions is determined by the :py:attr:`Transformer`."""
 
 
-class Parameters(_Commentable):
-    """Parameters to an :py:class:`HRF` model."""
-    PeakDelay: Optional[float]
-    """Delay, in seconds, from onset to peak response.
-    Applies to models: Gamma, DoubleGamma."""
-    PeakDispersion: Optional[float]
-    """Width of peak.
-    Applies to models: Gamma, DoubleGamma."""
-    UndershootDelay: Optional[float]
-    """Delay, in seconds, from onset to undershoot response.
-    Applies to model: DoubleGamma."""
-    UndershootDispersion: Optional[float]
-    """Width of undershoot.
-    Applies to model: DoubleGamma."""
-    PeakUndershootRatio: Optional[float]
-    """Peak-to-undershoot ratio.
-    Applies to model: DoubleGamma."""
-    Derivatives: Optional[int]
-    """Order of derivatives to include. 1 indicates the first derivative,
-    while 2 indicates the first and second derivative.
-    Applies to models: Gamma, DoubleGamma."""
-    Delays: Optional[List[int]]
-    """List of delays, in scans, for impulse responses.
-    Applies to model: FiniteImpulseResponse."""
-
-
 class HRF(_Commentable):
     """Specification of a hemodynamic response function (HRF) model."""
     Variables: List[str]
     """Name of the variables to be convolved. These must appear in :py:attr:`Model.X`"""
-    Model: HRFModel
-    """Name of a hemodynamic model."""
-    Parameters: Optional[Parameters]
-    """Parameters to the hemodynamic model."""
+    Model: str
+    """Name of a hemodynamic model.
+
+    Known model names include:
+
+    * ``"spm"``
+    * ``"spm + derivative"``
+    * ``"spm + derivative + dispersion"``
+    * ``"glover"``
+    * ``"glover + derivative"``
+    * ``"glover + derivative + dispersion"``
+    * ``"fir"``
```

Comment:
```suggestion
    * ``"afni"``
    * ``"fir"``
```

Reply:
While you could say that much more is needed to be elaborated here, adding "afni" is really just a placeholder, and who ever builds the BIDS-models to AFNI can refine it.

Reply:
If there is no better name ;-)
should there be 'afni + derivative' etc. then ?

Reply:
Interestingly, the `GAM` gamma HRF in AFNI's `3dDeconvolve` doesn't offer a derivative option; derivatives are only offered through the `SPMG2` and `SPMG3`.  I'm guessing that most AFNI users  use`SPMG1` or `SPMG2`, but `GAM` seems to be the recommended default in AFNI documentation and educational materials.